src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1640,6 +1640,12 @@ looseItem = do
   skipopts
   return mempty
 
+epigraph :: PandocMonad m => LP m Blocks
+epigraph = do
+  p1 <- grouped blocks
+  p2 <- grouped blocks
+  return $ divWith ("", ["epigraph"], []) (fromList $ concat $ map toList [p1, p2])
```

Comment:
Can't `fromList $ concat $ map toList [p1, p2]` be replaced with `p1 <> p2`?

Reply:
Definitely. Fixed!